### PR TITLE
PHP Object Injection Tests

### PR DESCRIPTION
**Pull request by rjt-gupta**

### CODE REVIEW

File: tanner/tests/test_php_object_injection.py [hunk 1]
```diff
@@ -0,0 +1,49 @@
...
+class TestPHPCodeInjection(unittest.TestCase):
+    def setUp(self):
+        self.loop = asyncio.new_event_loop()
+        asyncio.set_event_loop(None)
+        self.handler = PHPObjectInjection(loop=self.loop)
+        self.result = None
+        self.expected_result = None
+        self.returned_result = None
+
+    def test_scan(self):
```
Comment (rnehra01) [inline tanner/tests/test_php_object_injection.py]: Test with wrong regex.

**Comment by rjt-gupta** [comment]
> 
> 
> What about get-results method?

From PHPSandbox Helper?

**Comment by rnehra01** [comment]
https://github.com/mushorg/tanner/blob/master/tanner/emulators/php_object_injection.py#L14

Ravinder Nehra

**Comment by rjt-gupta** [comment]
> 
> https://github.com/mushorg/tanner/blob/master/tanner/emulators/php_object_injection.py#L14 

It is included in `test_handle`. Should I add separate test also?

**Comment by rnehra01** [comment]
Is it possible to do this without mocking phpox helper?

**Comment by rjt-gupta** [comment]
> 
> 
> Is it possible to do this without mocking phpox helper?

I don't think so, there is no default phpox right?

File: tanner/tests/test_php_object_injection.py (hunk 1 above)
Comment (rnehra01) [reply tanner/tests/test_php_object_injection.py]: ??

File: tanner/tests/test_php_object_injection.py [hunk 2]
```diff
@@ -21,6 +21,13 @@ def test_scan(self):
         self.returned_result = self.handler.scan(payload)
         self.assertEqual(self.returned_result, self.expected_result)
 
+    def test_scan_negative(self):
```
Comment (rjt-gupta) [inline tanner/tests/test_php_object_injection.py]: Here.

**Pull request merged**